src/Verifier/verifier.go

```diff
@@ -0,0 +1,102 @@
+package Verifier
+
+import (
+	"bufio"
+	"errors"
+	"os"
+	"slices"
+	"strings"
+)
+
+// This is for alternative names for blips e.g. CSharp, CS, C# all should be counted as C#)
+// var alt_names_str = `Go;golang,go-lang
+// 					C++;cpp
+// 					C#;csharp,cs
+// 					Python;python3,py
+// 					`
+
+
+func Verifier (filepaths ... string) error {
+
+	//TODO: Unmarshal the json file (or some other file based solution) to get the alternative names
+	// Or just use a baked in str read line by line or combination
+	//os.Stat("./Dictionary/alt_names.txt")
+
+	alt_names := make(map[string]string)//{"golang":"Go","go-lang:Go","cpp":"C++","csharp":"C#","cs":"C#","python3":"Python","py":"Python"}
+	alt_names["python3"] = "Python"
+
+
+
+	// Map functions as a set (name -> ring)
+	set := make(map[string][]string)
+
+	//TODO: This is where a for loop for multi csv should go
+	for _, filepath := range filepaths {
+	
+	file, err := os.Open(filepath)
+	if err != nil {
+		panic(err)
+	}
+
+	// Create temp file to overwrite primary file
+	tempfile, err := os.Create("tempfile.csv")
+	if err != nil {
+		panic(err)
+	}
+
+	defer file.Close()
+	defer tempfile.Close()
+	defer os.RemoveAll(tempfile.Name())
+
+	scanner := bufio.NewScanner(file)
+	
+	//TODO: Check if header matches (this will be another branch) for now it will just eat the first line
+	scanner.Scan()
+	tempfile.WriteString(scanner.Text()+"\n")
+	
+	// https://stackoverflow.com/questions/44073754/how-to-slice-string-till-particular-character-in-go
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		if line == "" {
+			break
+		}
+
+		// Faster than splitting
+		// Panic handler 
+		name := ""
+		index := strings.IndexByte(line, ',')
+		if index != -1 {
+		name = line[:index]
+		} 
+		
+		//TODO: Change these to be variable names later if we extend the program with custom ring names
+		if strings.Contains(name,"Hold") || strings.Contains(name,"Adopt") || strings.Contains(name,"Assess") || strings.Contains(name, "Trail") || name == "" {
+			os.Rename("tempfile.csv", filepath)
+			return errors.New("No comma was found format of csv file is wrong: triggered by line -> "+line)
```

Comment:
Why are we overwriting the original file with the tempfile if there was an error?

Reply:
Well since we are throwing an error, should that not be caught by the call, and then handled such that the merger doesn't get called with invalid files?

Reply:
That is a very good point I'll try and make that change

Reply:
Should be fixed now